### PR TITLE
Fix Klockwork compilation warning

### PR DESCRIPTION
**Pull request by chenhayat**

### CODE REVIEW

File: fmt/format.h [hunk 1]
```diff
@@ -1912,7 +1913,10 @@ class ArgFormatterBase : public ArgVisitor<Impl, void> {
 
   void visit_bool(bool value) {
     if (spec_.type_)
-      return visit_any_int(value);
+    {
+        visit_any_int(value);
+        return;
+    }
```
Comment (vitaut) [inline fmt/format.h]: Please follow Google style guide: https://google.github.io/styleguide/cppguide.html

**Comment by foonathan** [comment]
@vitaut Could you perhaps add a .clang-format file? It would make contributing much easier.

**Comment by vitaut** [comment]
@foonathan What will be the benefit of having `.clang-format` in the project itself compared to say passing `-style=google` to `clang-format`?

**Comment by foonathan** [comment]
Oh, so you're using the google style without modification? Did not know that.

**Comment by vitaut** [comment]
Yes, with the exception of snake_case instead of CapitalizedWords for method names, but this doesn't affect formatting. I should probably document it somewhere.

**Comment by foonathan** [comment]
Yes, you should document it.

But running clang-format over the source files will also change some other stuff like the indentation of your `#defines` when in `#ifdef`, break lines differently, etc.

**Comment by vitaut** [comment]
@chenhayat Merged in https://github.com/fmtlib/fmt/commit/05ba3e788846957ddb88277ed73b54073e595907, thanks!

@foonathan I've documented the coding style in https://github.com/fmtlib/fmt/blob/master/CONTRIBUTING.rst. `clang-format` does mess up preprocessor indentation, so I am a bit reluctant of applying it globally. Hopefully they'll fix https://llvm.org/bugs/show_bug.cgi?id=17362 and until then I suggest limiting it to formatting patches.

**Pull request closed without merging**